sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.8.1 (2023-01-13)
+
+### Bugs Fixed
+- Fixed an issue when using `ManagedIdentityCredential` in combination with authorities other than Azure public cloud that resulted in a bogus instance metadata validation error. [#32498](https://github.com/Azure/azure-sdk-for-net/issues/32498)
```

Comment:
Not sure who well "bogus" is understood or could be translated (though, I don't think we're doing that right now).
```suggestion
- Fixed an issue when using `ManagedIdentityCredential` in combination with authorities other than Azure public cloud that resulted in a incorrect metadata validation error. [#32498](https://github.com/Azure/azure-sdk-for-net/issues/32498)
```

Or maybe "an erroneous".